test/acceptance/create_webdriver.robot

```diff
@@ -15,38 +15,54 @@ Create Webdriver Creates Functioning WebDriver
 
 Create Webdriver With Bad Driver Name
     [Documentation]    Invalid browser name
-    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name
-    ...    Create Webdriver    Fireox
+    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name    Create Webdriver
+    ...    Fireox
```

Comment:
What was done? If this change has already been merged, it shouldn't anymore show in this PR. If this test has been changed after creating this PR, then this PR should be updated.